compiler/rustc_middle/src/mir/interpret/mod.rs

```diff
@@ -546,7 +547,7 @@ impl<'tcx> TyCtxt<'tcx> {
 
     /// Freezes an `AllocId` created with `reserve` by pointing it at an `Allocation`. May be called
     /// twice for the same `(AllocId, Allocation)` pair.
-    fn set_alloc_id_same_memory(self, id: AllocId, mem: &'tcx Allocation) {
+    pub fn set_alloc_id_same_memory(self, id: AllocId, mem: &'tcx Allocation) {
```

Comment:
Why does that need to be public now?

I don't know the details of this function, but I know there are some subtle invariants around this code, so making things more public is always suspicious. :)